Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN --mount=type=cache,target=/root/.cache/pip pip3 install --user --requirement
 
 # Build stage: Install yarn dependencies
 # ===
-FROM node:12-slim AS yarn-dependencies
+FROM node:12 AS yarn-dependencies
```

Comment:
@jkfran Is this correct solution or just a workaround?

Reply:
It's fine 👍🏽